packages/stryker-typescript/package.json

```diff
@@ -51,5 +51,17 @@
   "peerDependencies": {
     "stryker-api": "^0.11.0",
     "typescript": "^2.5.0"
+  },
+  "initStrykerConfig": {
+    "tsconfigFile": "tsconfig.json",
+    "files": [],
+    "coverageAnalysis": "off",
+    "mutator": "typescript",
+    "transpilers": [
+      "typescript"
```

Comment:
Overwriting this can cause issues when we start supporting webpack

Reply:
Yeah, i know. What else can we do? Maybe we should have slit it up into more packages? Or have an other way of opting in-out of initial config somehow?

Reply:
We could remove it here and rely on the initialiser to set the correct config for the transpilers array

Reply:
wow, is that also set by the initializer? Sorry, i didn't realize 👍